src/routes/TicketHistory.tsx

```diff
@@ -115,256 +189,240 @@ export function TicketHistoryRoute() {
+		}
+		if (selectedCell === "void" && !selectedData.void && ticketDate == todaysDate) {
+			setIsVoidOpen(true);
+			//resetting note if one got leftover somehow
+			setVoidNote("");
+			setVoidTicketId(selectedData.id);
+		} else if (
+			selectedCell != "note" &&
+			selectedCell != "image" &&
+			selectedCell != "signature" &&
+			selectedCell != "void" &&
+			selectedCell != "pdf" &&
+			selectedCell != "synced"
+		) {
+			setTicketPopupContent(ticketDetails(selectedData, udfs));
+			setIsTicketPopupOpen(true);
+		}
+	};
 
 	return (
 		<div className="flex h-full flex-col space-y-4">
 			<PageHeader title="Ticket History" />
 
-			{/* Date pickers */}
-			<div className="flex flex-row space-x-2">
-				<Input
-					type="date"
-					label="From date:"
-					value={fromDate}
-					onChange={(e) => setFromDate(e.target.value)}
-					containerClassName="min-w-[180px]"
-				/>
-				<Input
-					type="date"
-					label="To date:"
-					value={toDate}
-					onChange={(e) => setToDate(e.target.value)}
-					containerClassName="min-w-[180px]"
-				/>
-				{/* Lane Drop-Down Here */}
-			</div>
 			<Box fullHeight={true}>
-				{/* Quick Search Box */}
-				<div className="flex flex-row gap-1 space-x-2">
-					<span className="m-auto align-bottom text-sm font-medium">Search:</span>
-					<Input
-						placeholder="Search..."
-						value={quickSearch}
-						onChange={(e) => {
-							setQuickSearch(e.target.value);
+				<div className="flex flex-row gap-2">
+					<div className="flex flex-grow flex-col gap-1">
+						{/* Date pickers */}
+						<div className="flex flex-row justify-normal gap-1">
+							<span className="my-auto align-bottom text-sm font-medium">From Date:</span>
+							<Input
+								type="date"
+								value={fromDate}
+								onChange={(e) => setFromDate(e.target.value)}
+								containerClassName="min-w-[180px]"
+							/>
+							<span className="my-auto align-bottom text-sm font-medium">To Date:</span>
+							<Input
+								type="date"
+								value={toDate}
+								onChange={(e) => setToDate(e.target.value)}
+								containerClassName="min-w-[180px]"
+							/>
+							<span className="my-auto align-bottom text-sm font-medium">Lane:</span>
+							<Search
+								items={lanes ?? []}
+								value={laneFilter}
+								displayValue={laneFilter?.laneName ?? "All"}
+								onChange={(item) => setLaneFilter(item as SyncConfigLane)}
+								onClear={() => setLaneFilter(null)}
+								renderSize={31}
+								render={(lane) => (
+									<div className="flex flex-row space-x-2">
+										<div>{lane?.laneName}</div>
+										<div>-</div>
+										<div>{lane?.location?.locationDescription}</div>
+									</div>
+								)}
+								containerClass="flex-grow"
+							/>
+						</div>
+						{/* Quick Search Box */}
+						<div className="flex flex-row gap-1 ">
+							<span className="m-auto align-bottom text-sm font-medium">Search:</span>
+							<Input
+								inputRef={focusSearchRef}
```

Comment:
Can be removed in favor of `autofocus`.

```suggestion
								autofocus
```